src/flows/templates/types/share.ts

```diff
@@ -0,0 +1,13 @@
+export default register =>
+  register({
+    type: 'share',
+    init: accessID =>
+      fetch(`/api/share/${accessID}`)
```

Comment:
this is very slick. Nicely done

Reply:
Don't we have a generated function we can use here?